app/transformations/MemberPaymentsTransformer.scala

```diff
@@ -156,35 +157,46 @@ class MemberPaymentsTransformer @Inject()(
         }
       }
 
+      val normalise = (memberDetails: MemberDetails) => {
```

Comment:
If we change the task list not to distinguish between In progress and Completed, do we need to revert this change? (FYI @ccatosdevelopment )

Reply:
Yeah we would, this is just to fix the QA bug